packages/internals/src/scope.js

```diff
@@ -20,7 +20,7 @@ class Scope {
     } else if (this.outerScope) {
       return this.outerScope.get(key);
     } else {
-      throw "Not defined";
+      throw new Error("Not defined");
```

Comment:
Can you change this to `Error! ${key} is undefined!` or something like that?

Reply:
Fixed by d2e1278